.vscode/settings.json

```diff
@@ -1,4 +1,5 @@
 {
+  "dotnet-test-explorer.testProjectPath": "test/**/*Test.csproj",
```

Comment:
This helps avoid showing the benchmark tests in the VS Code test explorer so you can just "run all tests" without having to worry about eating the next 45 minutes with benchmarks.